.github/templates/pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+*Descriptive Summary of your Bugfix, Feature, or Refactoring.*
+
+## List of Added or Changed Components
+- feature a added to module bar
+- bug b fixed in foo.rs
+
+## Open Questions
+- [ ] should I go with solution A or B?
```

Comment:
Same here, in the interest of keeping things minimal I think this section can be dropped. I expect most PRs won't have open questions.